migrations/applied_migrations/LetterYesNo.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+echo ""
```

Comment:
these can be deleted, you dont need them, I suggest updating your migrate script to just delete rather than move